Orange/widgets/data/owcorrelations.py

```diff
@@ -402,6 +415,8 @@ def commit(self):
         corr_table.name = "Correlations"
 
         self.Outputs.data.send(self.data)
+        self.info.set_output_summary(len(self.data),
```

Comment:
To avoid duplicated code, I'd suggest removing upper to lines and add something like:

```
    def commit(self):
        self.Outputs.data.send(self.data)
        summary = len(self.data) if self.data else self.info.NoOutput
        detail = format_summary_details(self.data) if self.data else ""
        self.info.set_output_summary(summary, detail)

        if self.data is None or self.cont_data is None:
            self.Outputs.features.send(None)
            self.Outputs.correlations.send(None)
            return
```

at the beginning of the function.